tingbot/platform_specific/sdl_wrapper.py

```diff
@@ -49,6 +49,7 @@ def click(self, xy, action):
             self.surface.fill(self.color)
             if button_callback:
                 button_callback(self.number, 'up')
+        screen.needs_update = True
 
```

Comment:
self.surface does not have an update method. However, you can call pygame.display.update with the button rect and this will just update that bit of it. I've added this to the pull request